west.yml

```diff
@@ -0,0 +1,45 @@
+# The west manifest file for nRF Connect SDK, NCS.
+#
+# The per-installation west configuration file, .west/config, sets the
+# path to the project containing this file in the [manifest] section's
+# "path" variable.
```

Comment:
Not sure I understand this paragraph. The config file sets the PATH. It adds the current project to it. And it determines what the path to the current project is by reading the manifest section - what in there? self:path?

Reply:
This sentence comes from upstream manifest, but we should of course make it more readable if that is needed.

What it tries to describe, is that the configuration file: `.west/config` will contain a setting `path = <folder>` and that `<folder>` will be the folder where this `west.yml` file is located.
In the NCS case, that will be:
```
[manifest]
path = nrf
```
I will be happy to update the description if you have any suggestions.
If I need to describe this better myself, I could end-up making it more unreadable .

Reply:
How about:

The per-installation west configuration file specifies the location of this manifest file. The "path" option in the [manifest] section of .west/config defines the folder that contains west.yml.

Reply:
Path relative to what, actually?

Reply:
The folder containing `.west`

Reply:
> The per-installation west configuration file specifies the location of this manifest file. The "path" option in the [manifest] section of .west/config defines the folder that contains west.yml.

Sounds better.

Reply:
OK, then:

The per-installation west configuration file specifies the location of this manifest file. The "path" option in the [manifest] section of .west/config defines the folder that contains west.yml. The path is relative to the folder that contains .west.

Reply:
Thanks a lot.
Much better and clearer than anything I could have written :)